client/app/shared/modals/model-delete/model-delete.component.ts

```diff
@@ -39,9 +42,12 @@ export class ModelDelete {
     })
   }
 
+  open () {
+    jQuery('#delete-modal').modal().modal('show')
+  }
+
   close () {
-    let closeButton: HTMLElement = document.querySelector('#model-delete [data-close]')
-    closeButton.click()
+    jQuery('#delete-modal').modal().modal('hide')
```

Comment:
You might not need `jQuery`: http://youmightnotneedjquery.com/

Reply:
This is actually a semantic function... https://semantic-ui.com/modules/modal.html